TechDoc-Assistance-Program.md

```diff
@@ -0,0 +1,55 @@
+# Technical Documentation Assistance Program
+
+This document outlines the Cloud Native Computing Foundation (CNCF) Technical Documentation Assistance Program (the Program), a service offered by CNCF Tech Docs for evaluating and improving an OSS project's technical documentation. The process is designed to:
+
+1. Provide a baseline analysis of the project's documentation quality measured against the project's [maturity level](analysis/analysis-tools/criteria.md. (Often, documentation help is requested when a project requests promotion to a new maturity level.)
+1. Recommend changes that will reduce the gap between the documentation and the maturity of the overall project.
+1. Expand on the recommended changes in an implementation plan.
+1. Break down the implementation into a documentation project backlog comprising a GitHub Issues list.
+1. Support the project team's contributors in taking up and completing the issue backlog items.
+1. Leave the project team with an improved understanding and skill base for improving and maintaining the project documentation.
+
+## Phase 0: Training
+
+Some level of familiarity with the technical documentation process is required to:
+
+- Work effectively with technical writers
+- Draft technical documentation (for non-writers)
+- Improve the chances of a good outcome from the Tech Doc Assistance Program
+
+For this reason, CNCF offers free training on documentation essentials for project contributors and maintainers. To get the most from the Technical Documentation Assistance Program, project contributors should do the training *before* engaging a documentation specialist to complete the documentation analysis.
```

Comment:
```suggestion
For this reason, CNCF offers free training on documentation essentials for project contributors and maintainers. To get the most from the Assistance Program, project contributors are encouraged to do the training *before* engaging a documentation specialist to complete the documentation analysis.
```

Should the last part be simplified to this:

```suggestion
For this reason, CNCF offers free training on documentation essentials for project contributors and maintainers. To get the most from the Assistance Program, project contributors should do the training *before* applying for the program.
```

Reply:
That seems like a good edit.